lib/src/contributors/class/shorthand_constructor.dart

```diff
@@ -86,14 +79,37 @@ class ShorthandConstructorAssistContributor extends Object
     addAssist(AvailableAssists.shorthandConstructor, changeBuilder);
   }
 
-  void _writeConstructor({
+  static void writeEmptyConstructor({
```

Comment:
Probably you can merge with with `writeConstructor` and check if the fields are empty then write empty constructor and return early

Reply:
If you want to keep that just remove the return on line 92